lib/cfn_nag.rb

```diff
@@ -206,9 +206,11 @@ def generic_json_rules(input_json, rule_directories)
     rule_directories.each do |rule_directory|
       rules = Dir[File.join(rule_directory, '*.rb')].sort
 
-      rules.each do |rule_file|
-        @input_json = input_json
-        eval IO.read(rule_file)
+      if rules.length > 0
```

Comment:
You know what, you're right. I think my brain wasn't caffeinated enough when I did this. I'll look deeper into what's causing that error.